src/citrine/_session.py

```diff
@@ -34,24 +33,15 @@ class Session(requests.Session):
     """Wrapper around requests.Session that is both refresh-token and schema aware."""
 
     def __init__(self,
+                 *,
                  refresh_token: str = None,
```

Comment:
```suggestion
                 refresh_token: str = None,
                 *,
```
Not that it necessarily makes sense, but the deprecation warning we have been issuing is that refresh_token can still be positional.

Reply:
Good point, I'll make the refresh token/api key positional